tests/drone/config.json

```diff
@@ -8,7 +8,7 @@
 		"authUrl": "http://owncloud/index.php/apps/oauth2/authorize"
 	},
 	"apps" : [
-		"files", "markdown-editor", "pdf-viewer", "media-viewer", "draw-io"
```

Comment:
this will have impact on the build command ... I'd love to keep this list.

Reply:
even if the other apps are not tested (yet)?